.env.example

```
@@ -5,5 +5,8 @@ GROQ_API_KEY=
 NVIDIA_NIM_API_KEY=
 MISTRAL_API_KEY=
 CLOUDFLARE_API_KEY=
-GEMINI_API_KEY=
+# מהדשבורד של Cloudflare — ‏Workers & Pages, בצד ימין
+CLOUDFLARE_ACCOUNT_ID=
+GEMINI1_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GEMINI1_API_KEY key should go before the GROQ_API_KEY key

(UnorderedKey)